packages/features/timezone-buddy/components/AvailabilitySliderTable.tsx

```diff
@@ -0,0 +1,165 @@
+
+import dayjs from "@calcom/dayjs";
+import type { DateRange } from "@calcom/lib/date-ranges";
+import type { MembershipRole } from "@calcom/prisma/enums";
+import { trpc } from "@calcom/trpc";
+import { Avatar, Button, ButtonGroup, DataTable } from "@calcom/ui";
+
+import { TBContext, createTimezoneBuddyStore } from "../store";
+import { TimeDial } from "./TimeDial";
+
+export interface SliderUser {
+  id: number;
+  username: string | null;
+  email: string;
+  timeZone: string;
+  role: MembershipRole;
+  dateRanges: DateRange[];
+}
+
+export function AvailabilitySliderTable() {
+  const tableContainerRef = useRef<HTMLDivElement>(null);
+  const [browsingDate, setBrowsingDate] = useState(dayjs());
+
+  const { data, isLoading, fetchNextPage, isFetching } = trpc.viewer.availability.listTeam.useInfiniteQuery(
+    {
+      limit: 10,
+      loggedInUsersTz: dayjs.tz.guess(),
+      startDate: browsingDate.startOf("day").toISOString(),
+      endDate: browsingDate.endOf("day").toISOString(),
+    },
+    {
+      getNextPageParam: (lastPage) => lastPage.nextCursor,
+      keepPreviousData: true,
+    }
+  );
+
+  const memorisedColumns = useMemo(() => {
+    const cols: ColumnDef<SliderUser>[] = [
+      {
+        id: "member",
+        accessorFn: (data) => data.email,
+        header: "Member",
+        cell: ({ row }) => {
+          const { username, email, timeZone } = row.original;
+          return (
+            <div className="max-w-64 flex flex-shrink-0 items-center gap-2 overflow-hidden">
+              <Avatar
+                size="sm"
+                alt={username || email}
+                imageSrc={"/" + username + "/avatar.png"}
+                gravatarFallbackMd5="fallback"
+              />
+              <div className="">
+                <div
+                  className="text-emphasis max-w-64 truncate text-sm font-medium leading-none"
+                  title={email}>
+                  {username || "No username"}
+                </div>
+                <div className="text-subtle text-sm leading-none">{timeZone}</div>
+              </div>
+            </div>
+          );
+        },
+      },
+      {
+        id: "timezone",
+        accessorFn: (data) => data.timeZone,
+        header: "Timezone",
+        cell: ({ row }) => {
+          const { timeZone } = row.original;
+          const timeRaw = dayjs().tz(timeZone);
+          const time = timeRaw.format("HH:mm");
+          const utcOffsetInMinutes = timeRaw.utcOffset();
+          const hours = Math.abs(Math.floor(utcOffsetInMinutes / 60));
+          const minutes = Math.abs(utcOffsetInMinutes % 60);
+          const offsetFormatted = `${utcOffsetInMinutes < 0 ? "-" : "+"}${hours
+            .toString()
+            .padStart(2, "0")}:${minutes.toString().padStart(2, "0")}`;
+
+          return (
+            <div className="flex flex-col gap-2 leading-none">
+              <span className="text-default text-sm font-medium">{time}</span>
```

Comment:
[Not a big issue] I have 12 hour time format selected but i still see everything 24 hour format.

WTB supports 12 hour format 

<img width="787" alt="Screenshot 2023-08-16 at 4 20 17 PM" src="https://github.com/calcom/cal.com/assets/53316345/1002e2c1-cc06-4d14-bbfb-e4087bff30b1">

Reply:
Will cover this in a follow up i think. We can do with 24h support for now. It requires some weirdness with dates and displaying that we need to figure out. @Jaibles we should probably figure out how we're going to display this. Likely the same approach as WTB also